knowledge-base/vod.mdx

```diff
@@ -0,0 +1,67 @@
+<Card title="What is the playback format for VOD?">
+Diagnose: The playback format is HLS and for any asset that is < 2 minutes is MP4.
```

Comment:
@suhailkakar, this was mentioned before WebRTC was the default. Could you confirm that playbacks for VOD is also defaulted to WebRTC? If it is, then can you change it, if not then we can leave it as is.